org.gnome.Nostalgia.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id" : "org.gnome.Nostalgia",
```

Comment:
If that is indeed a policy we really need to change Builder to allow setting a custom RDNN ID when you start a new project from a template. Currently there's no easy way to set an ID that isn't org.gnome.Appname there.

I guess in this specific case it might be ok to use org.gnome either way though, since it's very directly GNOME-related :)

Reply:
Yeah, I hoped to inspire @TingPing and @nedrichards what are their current thoughts on that these days. I don't mind it in this case personally.

Reply:
FYI https://gitlab.gnome.org/GNOME/gnome-builder/issues/720

Reply:
The issue was more to do with the upstream GNOME policy for their namespace and the difficulty of changing app-ids.

Reply:
So should I change it?

Reply:
Hey, 

In general you need permission to use the GNOME trademark... but well, it's a bit hypothetical here I think.

Whether software hosted in gitlab.gnome.org but not present in the gitlab.gnome.org/GNOME group can use GNOME trademark or not is still a bit of a gray area and the default answer should be that they need explicit permission. We are discussing a more automatic approach, but it's still not there.

My advice would be to either remove the GNOME prefix or to send an email to licensing@gnome.org, they can give approvals without much hassle for the regular case.

Let me know if you have more questions!

Reply:
Hmm then I'll just change the ID to im.bernard.Nostalgia, no need to bother even more people with my stupid side project :D

Reply:
It's not a bother, that's why the email exists :P we have quite a few of them, including toy projects, asking for permission there. Feel free to do any of those two things.

Reply:
@bertob Any news? Just want to know whether you are going to request trademark permission or rename the app, can't wait to use older wallpaper. :smile:

Reply:
I decided to change the domain to my own, it makes a lot more sense for a side project like this. I've tagged 0.2 and will make a new PR with that soon :)